llvm-build/libllvm.ps1

```diff
@@ -0,0 +1,131 @@
+Write-Vso "Test Linux Path: $linuxPath"
+Test-Path $linuxPath
+
+$windowsPath = Resolve-Path (Join-Path  '.' "$($env:WINDOWS_PKG_NAME)" "bin" "LLVM-C.dll")
+Write-Vso "Test Windows Path: $windowsPath"
+Test-Path $windowsPath
+
+$darwinPath = Resolve-Path (Join-Path  '.' "$($env:DARWIN_PKG_NAME)" "lib" "libLLVM.dylib")
+Write-Vso "Test Darwin Path: $darwinPath"
+Test-Path $darwinPath
+
+mkdir $env:PKG_NAME | Out-Null
+$packageDir = Resolve-Path "$($env:PKG_NAME)"
+
+Write-Vso "Move-Item -Path $linuxPath -Destination $(Join-Path $packageDir 'libLLVM.so')"
+Move-Item -Path $linuxPath -Destination (Join-Path $packageDir "libLLVM.so")
+
+Write-Vso "Move-Item -Path $windowsPath -Destination $(Join-Path $packageDir 'libLLVM.dll')"
+Move-Item -Path $windowsPath -Destination (Join-Path $packageDir "libLLVM.dll")
+
+Write-Vso "Move-Item -Path $darwinPath -Destination $(Join-Path $packageDir 'libLLVM.dylib')"
+Move-Item -Path $darwinPath -Destination (Join-Path $packageDir "libLLVM.dylib")
+
+$libLlvmPackage = Join-Path "$($env:BUILD_ARTIFACTSTAGINGDIRECTORY)" "$($env:PKG_NAME).zip"
+$compress = @{
+    Path             = $packageDir
+    CompressionLevel = "Fastest"
+    DestinationPath  = $libLlvmPackage
+}
+Compress-Archive @compress
+
+# Create LlvmBindings.Interop wrappers
+Write-Vso "Creating Interop Bindings"
+$includeDir = Join-Path  '.' "$($env:LINUX_PKG_NAME)" "include"
+$generatedDir = Join-Path $env:BUILD_ARTIFACTSTAGINGDIRECTORY generated
+mkdir $generatedDir | Out-Null
+dotnet tool install --global ClangSharpPInvokeGenerator --version 13.0.0-beta1
+ClangSharpPInvokeGenerator `
+    "@$(Join-Path $PSScriptRoot GenerateLLVM.rsp)" `
+    --output $generatedDir `
+    --file-directory $includeDir `
+    --include-directory $includeDir
+
+mkdir (Join-Path $PSScriptRoot drops) | Out-Null
+
+Write-Vso "Copy-Item -Path $(Join-Path $packageDir 'libLLVM.so') -Destination $(Join-Path $PSScriptRoot drops 'libLLVM.so')"
+Copy-Item -Path $(Join-Path $packageDir 'libLLVM.so') -Destination (Join-Path $PSScriptRoot drops "libLLVM.so")
+
+Write-Vso "Copy-Item -Path $(Join-Path $packageDir 'libLLVM.dll') -Destination $(Join-Path $PSScriptRoot drops 'libLLVM.dll')"
+Copy-Item -Path $(Join-Path $packageDir 'libLLVM.dll') -Destination (Join-Path $PSScriptRoot drops "libLLVM.dll")
+
+Write-Vso "Copy-Item -Path $(Join-Path $packageDir 'libLLVM.dylib') -Destination $(Join-Path $PSScriptRoot drops 'libLLVM.dylib')"
+Copy-Item -Path $(Join-Path $packageDir 'libLLVM.dylib') -Destination (Join-Path $PSScriptRoot drops "libLLVM.dylib")
+
+Write-Vso "Copy-Item -Recurse -Verbose -Path $generatedDir -Destination $(Join-Path $PSScriptRoot drops)"
+Copy-Item -Recurse -Verbose -Path $generatedDir -Destination $(Join-Path $PSScriptRoot drops)
+
+foreach ($file in $llvmArchiveFiles) {
+    Move-Item -Path $file -Destination $artifactsDir
+}
+
+# Add it the job's artifacts
+Copy-Item -Path $libLlvmPackage -Destination $artifactsDir
```

Comment:
Is something missing in the comment? 
"# Add it **to** the job's artifacts"?

(Also, I personally prefer when the use of pronouns (like "it", "there", "him") is _minimized_, and actual nouns are used instead (e.g. "# Add **the package** the job's artifacts" ). Because with pronouns the multiple interpretations are possible quite often)

Reply:
Yeah, that's good advice. The "it" here is indeed the package.